index.bs

```diff
@@ -2857,14 +2857,48 @@ Note: The {{CollectedClientData}} may be extended in the future. Therefore it's
 
     The {{CollectedClientData}} structure is used by the client to compute the following quantities:
 
-    : <dfn dfn>JSON-serialized client data</dfn>
-    :: This is the result of [=serialize JSON to bytes|JSON-serializing to bytes=] a
-        {{CollectedClientData}} dictionary.
+    : <dfn dfn>JSON-compatible serialization of client data</dfn>
+    :: This is the result of performing the [JSON-compatible serialization algorithm](#clientdatajson-serialization) on the {{CollectedClientData}} dictionary.
 
     : <dfn dfn>Hash of the serialized client data</dfn>
-    :: This is the hash (computed using SHA-256) of the [=JSON-serialized client data=], as constructed by the client.
+    :: This is the hash (computed using SHA-256) of the [=JSON-compatible serialization of client data=], as constructed by the client.
 </div>
 
+#### Serialization #### {#clientdatajson-serialization}
+
+The serialization of the {{CollectedClientData}} is a subset of the algorithm for [=serialize JSON to bytes|JSON-serializing to bytes=]. I.e. it produces a valid JSON encoding of the {{CollectedClientData}} but also provides additional structure that may be exploited by verifiers to avoid integrating a full JSON parser.
```

Comment:
what explicitly is the "additional structure" and how might it help a verifier?

I'd s/ JSON encoding / JSON serialization /

Reply:
Explicitly defined order of keys, I think. Allows for fewer loops and conditional branches.

Reply:
Have enumerated the requirements for the limited verification algorithm.